lib/rubocop/cop/rspec/predicate_matcher.rb

```diff
@@ -150,11 +150,20 @@ def check_explicit(node) # rubocop:disable Metrics/MethodLength
 
           predicate_matcher?(node) do |actual, matcher|
             add_offense(node, message: message_explicit(matcher)) do |corrector|
+              next if uncorrectable_matcher?(node, matcher)
+
               corrector_explicit(corrector, node, actual, matcher, matcher)
             end
           end
         end
 
+        def uncorrectable_matcher?(node, matcher)
+          return false unless (arg = matcher.first_argument)
```

Comment:
Does it have to be the first argument?
What about
```ruby
expect(foo.include?('one', <<~TWO))
  ...
TWO
```

Reply:
Thank you! I updated this PR.